Userland/Libraries/LibWeb/Painting/MediaPaintable.cpp

```diff
@@ -299,8 +300,8 @@ MediaPaintable::DispatchEventOfSameName MediaPaintable::handle_mouseup(Badge<Eve
 
     if (was_tracking_mouse) {
         if (was_tracking_timeline) {
-            set_current_time(media_element, *cached_layout_boxes.timeline_rect, position, Temporary::No);
-            media_element.set_layout_display_time({}, {});
+            set_current_time(media_element, *cached_layout_boxes.timeline_rect, position);
```

Comment:
This call to `set_current_time` can likely be removed? We'll have issued a committed time on the last mouse move I think?

Reply:
In my experience (granted, not in Qt or LibGUI), the mouseup event can trigger with a different mouse position than the last mousemove, so I tend to prefer acting on the mouse position provided to a mouseup event to avoid ending up with a stale value.

Reply:
Fair!